spec/api/holdings_api_spec.rb

```diff
@@ -232,8 +232,62 @@ def parse_body(call)
         build(:holding, organization: "umich", ocn: ocn),
         build(:holding, organization: "smu", ocn: ocn)
       )
-      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1, organization: "umich")))
+      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1)))
       expect(response["organizations"].sort).to eq ["smu", "umich"]
     end
   end
+
+  describe "v1/item_held_by_brlm" do
+    it "returns an application error if the item_id does not match an ht_item" do
+      response = parse_body(v1(url_template("item_held_by_brlm", item_id: item_id_1)))
+      expect(response["application_error"]).to eq "no matching data"
+    end
+    it "reports nothing if there are no holdings" do
+      load_test_data(htitem_1)
+      response = parse_body(v1(url_template("item_held_by_brlm", item_id: item_id_1)))
+      expect(response["organizations"]).to eq []
+    end
+    it "reports nothing if all holdings are status:CH & condition:''" do
+      load_test_data(htitem_1)
+      load_test_data(build(:holding, organization: "a", ocn: htitem_1.ocns.first, status: "CH", condition: ""))
+      response = parse_body(v1(url_template("item_held_by_brlm", item_id: item_id_1)))
+      expect(response["organizations"]).to eq []
+    end
+    it "reports nothing if all holdings are status:WD & condition:''" do
```

Comment:
All the tests here make sense, but it may be ultimately preferable to move some of them either to tests for `Clusterable::Holding#brt_lm_access?` or to `Cluster#access_counts`. (Right now it doesn't appear we have any specific tests for those, so it makes sense to be comprehensive here.)

Reply:
added test for `Clusterable::Holding#brt_lm_access?`

Reply:
added tests for `Cluster#access_counts`